### ISSUE TITLE
API Key Unauthorized Again

### ISSUE THREAD

**Issue by pbortnick** [contra/spotcrime#14]
I saw that there was an issue with the API key in the past, and I believe it is back.

I am getting:
`Unhandled rejection Error: Unauthorized`

when processing the example

```
const spotcrime = require('spotcrime');

const loc = {
      lat: 33.39657,
      lon: -112.03422
    };
    
const radius = 0.01;

spotcrime.getCrimes(loc, radius).then((crimes) => {
     debugger
});
```

thanks!

**Comment by contra** [comment]
@contra Wasn't able to quickly find the new key so I'm going to put a notice in the README instead, and I'll look into it again later.

I would recommend looking at https://municipal.systems/ if you need an API for crime data (or any other municipal data).

**Comment by contra** [comment]
Ah I was able to find it again, and added a note to the README about future breakage. Published as 2.0.1.

**Comment by pbortnick** [comment]
Just came here to tell you I found it too. Seems like its changes often :/

Thanks for the tip. Will checkout that site.
